src/checkers/rustchecker.py

```diff
@@ -31,13 +36,21 @@ def check(self, external_data):
 
         package = data["pkg"][package_name]
         target = package["target"][target_name]
+
+        release_date = datetime.date.fromisoformat(data["date"])
+        version, _, _ = VERSION_RE.match(package["version"]).groups()
+        if channel == "stable":
```

Comment:
this case should probably include the `beta` channel
the versions look like this `1.49.0-beta.5 (b0dc3c6f4 2020-12-22)` so using `1.49.0-beta.5` should be fine

Reply:
Exactly `beta` or any non-`nightly`?

Reply:
any non nightly seems best to me